### DISCUSSION TITLE
How to add this repository as a nix channel - lack of nixexprs.tar.bz2

### DISCUSSION THREAD

**Discussion by rsniezek** [nix-community/kde2nix#38] [Q&A] [answered]
Hi all,

In the readme it states that on non-flake system I should add this repo as a channel. However, doing 
```bash
nix-channel --add https://github.com/nix-community/kde2nix
nix-channel --udpate kde2nix
```
results with
```bash
error: unable to download 'https://github.com/nix-community/kde2nix/nixexprs.tar.bz2': HTTP error 404

       response body:

       Not Found
```
am I missing something obvious? I cannot go flake route as my configuration.nix has some options that are not recognized when switching to flake setup (story for another time).
Can we update the readme so that it contains more detailed instructions for people with lesser nix experience as myself?

Thanks,
Radek

**Comment by K900** [comment, marked as answer]
You want `https://github.com/nix-community/kde2nix/archive/main.tar.gz` as the URL.

**Reply by rsniezek** [reply]
Thank you, it does work. Created PR to update the readme: https://github.com/nix-community/kde2nix/pull/39
